zerver/actions/realm_settings.py

```diff
@@ -552,3 +560,73 @@ def do_send_realm_reactivation_email(realm: Realm, *, acting_user: Optional[User
         language=language,
         context=context,
     )
+
+
+def do_send_realm_deactivation_email(realm: Realm, acting_user: Optional[UserProfile]) -> None:
+    deactivation_date = timezone_now().date()
+    shared_context = {
+        "realm_name": realm.name,
+        "event_date": deactivation_date,
+        "corporate_enabled": settings.CORPORATE_ENABLED,
+    }
+    language = realm.default_language
+    owner_emails = set(realm.get_human_owner_users().values_list("delivery_email", flat=True))
+
+    not_deactivated_by_owner = False
+
+    # This is an optional parameter for the deactivate_realm management command;
+    # there is no deactivating user name for the email to owners.
+    if acting_user is None:
+        not_deactivated_by_owner = True
+
+    # This is realm deactivation is from the support panel;
+    # we do not share the deactivating user's name in this case.
+    if acting_user is not None and acting_user.delivery_email not in owner_emails:
+        not_deactivated_by_owner = True
+
+    if not_deactivated_by_owner:
+        not_owner_context = shared_context.copy()
+        not_owner_context["no_acting_user"] = True
+        send_email(
+            "zerver/emails/realm_deactivated",
+            to_emails=list(owner_emails),
+            from_name=FromAddress.security_email_from_name(language=language),
+            from_address=FromAddress.tokenized_no_reply_address(),
+            language=language,
+            context=not_owner_context,
+            realm=realm,
+        )
```

Comment:
> This flag is optional for the deactivate_realm management command, but as there is no active user passed in that case, then the email is sent without a reference to who deactivated the realm.

If desired think we could say "The {EXTERNAL_HOST} administrators" as who deactivated this organization. I think in some sense for Zulip Cloud we may want this to say something like "Zulip support" or "Zulip account" as who did the deactivation if done via management command.

Reply:
I guess we already have "Zulip account security" as the email sender.

Reply:
I didn't change the text of the emails for anonymous deactivation via support or management command. See update and updated screenshots below.

Reply:
Works for now, we can always do that in a follow-up.